### PR TITLE
Fixed an issue where user mfa entry was not getting delete after deleting a user. #5833

### PR DESCRIPTION
**Pull request by pravesh-sharma**

### CODE REVIEW

File: web/pgadmin/tools/user_management/__init__.py [hunk 1]
```diff
@@ -597,6 +597,9 @@ def delete_user(uid):
         SharedServer.query.filter(SharedServer.servergroup_id.in_(sg)).delete(
             synchronize_session=False)
 
+        # Delete MFA setting for current user.
+        UserMFA.query.filter_by(user_id=uid).delete()
```
Comment (adityatoshniwal) [inline web/pgadmin/tools/user_management/__init__.py:601]: Shouldn't this be a delete cascade? user id must be having a FK on user table. So it should get deleted automatically on delete in user table.

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]
The review was marked resolved without any change.

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]
Please use FK reference with delete cascade.

File: web/pgadmin/tools/user_management/__init__.py [hunk 2]
```diff
@@ -597,6 +597,9 @@ def delete_user(uid):
...
+
```
Comment (adityatoshniwal) [inline web/pgadmin/tools/user_management/__init__.py:602]: Use foreign key.

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]

File: web/pgadmin/tools/user_management/__init__.py (hunk 2 above)
Comment (pravesh-sharma) [reply web/pgadmin/tools/user_management/__init__.py]: Made the required changes, please review.

**Review by adityatoshniwal — APPROVED** [review]

**Pull request merged**